src/material/schematics/ng-generate/navigation/files/__path__/__name@dasherize@if-flat__/__name@dasherize__.component.html.template

```diff
@@ -5,9 +5,9 @@
       [opened]="(isHandset$ | async) === false">
     <mat-toolbar>Menu</mat-toolbar>
     <mat-nav-list>
-      <a mat-list-item href="#">Link 1</a>
-      <a mat-list-item href="#">Link 2</a>
-      <a mat-list-item href="#">Link 3</a>
+      <a mat-list-item <% if(routing) routerLink="/" %> <% if(!routing) href="#" %> >Link 1</a>
+      <a mat-list-item routerLink="/">Link 2</a>
```

Comment:
Don't we need `if(routing)` checks on these two links as well? Also you might be able to use a ternary here so it's all in the same expression.

Reply:
Done.